MTA-6893 Document Hub OIDC authentication (#382)
migtools/mta-documentation: assemblies/mta-install-title/assembly_red-hat-build-of-keycloak.adoc

:context: red-hat-build-of-keycloak

[role="_abstract"]

> The {ProductFullName} Hub OpenID Connect supports integration of a Red Hat build of Keycloak (RHBK) instance for user authentication and authorization for users who upgrade to {ProductShortName} 8.2.0.  

Reviewer: ```suggestion
When you upgrade to {ProductShortName} 8.2.0, you can integrate a Red Hat build of Keycloak (RHBK) with the {ProductFullName} Hub OpenID Connect to authenticate and authorize users.
```
